.buildkite/pipeline.ml.yml

```diff
@@ -290,7 +290,7 @@
   commands:
     - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/travis/upload_build_info.sh; fi }; trap cleanup EXIT
     - TRAIN_TESTING=1 INSTALL_HOROVOD=1 ./ci/travis/install-dependencies.sh
-    - bazel test --config=ci $(./scripts/bazel_export_options) --build_tests_only --test_tag_filters=-gpu_only,-tune python/ray/train/...
+    - bazel test --config=ci $(./scripts/bazel_export_options) --build_tests_only --test_tag_filters=-gpu_only,-minimal,-tune python/ray/train/...
```

Comment:
Do we still need to have the tune label/separating tune tests vs. other train tests? The minimal install should already  catch unintended tune imports as well.

Reply:
I don't have a super strong opinion here, but I think it's nice to still split it so you can run the tests in parallel and it helps isolating the problem easier if it fails in one suite but not the other.